charts/partials/templates/_pull-secrets.tpl

```diff
@@ -1,6 +1,4 @@
-{{- define "partials.image-pull-secrets" -}}
-{{- if .Values.global.imagePullSecrets }}
+{{- define "partials.image-pull-secrets"}}
 imagePullSecrets:
-{{ toYaml .Values.global.imagePullSecrets | indent 2 }}
-{{- end }}
+{{ toYaml . | indent 2 }}
 {{- end -}}
```

Comment:
Can you leave the boilerplate in the partial like it used to be, to avoid repeating it on each calling side?

Reply:
Done!